Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,38 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    /// See https://curl.haxx.se/docs/sslcerts.html
+    /// For SSL to work you need a "cacert.pem" to be accessible at the
+    /// by setting URLSession.sslCertificateAuthorityFile to a file URL.
+    /// Downloadable here: https://curl.haxx.se/docs/caextract.html
+    /// Alternatively you can bypass SSL peer verification altogether:
+    /// URLSession.sslUnsafeBypassPeerVerify = true (not advised)
+    fileprivate static var _sslCertificateAuthorityFile: UnsafeMutablePointer<Int8>?
+    public static var sslUnsafeBypassPeerVerify = false
+    public static var sslCertificateAuthorityFile: URL {
+        set(value) {
+            sslUnsafeBypassPeerVerify = false
+            free(_sslCertificateAuthorityFile)
+            guard value.isFileURL && FileManager.default.fileExists(atPath: value.path) else {
+                NSLog("*** sslCertificateAuthorityFile not FileURL or does not exist ***")
+                _sslCertificateAuthorityFile = nil
+                return
+            }
+            value.path.withCString {
+                _sslCertificateAuthorityFile = strdup($0)
+            }
+        }
+        get {
+            if let value = _sslCertificateAuthorityFile {
+                return URL(fileURLWithPath: String(cString: value))
+            }
+            else {
+                return URL(fileURLWithPath: "NOVALUE")
```

Comment:
Huh? There is a standard way to indicate the absence of something in Swift, and that's `nil`. This computed property should be of type `URL?`, and here the return value should be `nil`.